public/js/toast.js

```diff
@@ -1,25 +1,54 @@
+const timeUntilFade = 4000;
+const fadeInterval = 8;
+const removeAtOpacity = 0.05;
+const maxOpacity = 1;
+const removePerInterval = 0.01;
+
 function createToast(text, type) {
-    const toast = document.createElement('div');
-    toast.classList.add('norefresh-toast');
-    toast.classList.add(type);
+    const toastContainer = document.createElement('div');
+    toastContainer.classList.add('.norefresh-toast');
 
-    const textElement = document.createElement('p');
-    textElement.textContent = text;
+    let toastTypeClass = '';
+    switch (type)
+    {
+        case 'success':
+            toastTypeClass = 'toast-success';
+            break;
+        case 'error':
+            toastTypeClass =  'toast-error';
+            break;
+        case 'warning':
+            toastTypeClass =  'toast-warning';
+            break;
+        default:
+            toastTypeClass =  'toast-info';
+            break;
```

Comment:
Waarom doe je het niet zo?
```suggestion
    let toastTypeClass = 'toast-' + type;
```